examples/database-user/ldapUser.json

```diff
@@ -83,15 +83,5 @@
         ]
       }
     }
-  },
-  "Outputs": {
-    "MongoDBIamUsername": {
-      "Value": {
-        "Fn::GetAtt": [
-          "ldapUser",
-          "Username"
-        ]
```

Comment:
the reason why we added this section was to show customers how they could output information about the cfn resource. I don't think we should remove it

Reply:
I understand the reasoning, but as of now, the examples, when creating the resources, fail with the error specified in the PR description. I think the error can lead to a lot of confusion for customers.

Reply:
I think for sure it's better to have a working example. In this case I understand that it's interesting to show something in the Outputs, but Username is a param provided by the user so they already know its value. And in this resource I think they provide all values so there anything interesting to be returned by the resource which might be useful in the output, I think

Reply:
I think a great example to keep a param in ouputs is in iam-aws example, UserCFNIdentifier that is calculated in the resource.